frontend/src/pages/SettingsPage.js

```diff
@@ -1,19 +1,57 @@
-import React from "react";
-import Button from "@material-ui/core/Button";
-import TypeWarningSetting from "../components/settings/TypeWarningSetting";
+import React, {useState} from "react";
+// import { getSettings } from '../services/settingsApi'
+import SettingsForm from "../components/settings/SettingsForm";
 
 export default function SettingsPage() {
+    const [settings, setSettings] = useState({
+        warningCategory: {
+            warningA: true,
+            warningD: true,
+        },
+        distance: 15,
+        route: {
+            start: "",
+            end: "",
+            points: []
+        }
+    })
+
+    const [isLoading, setIsLoading] = useState(false);
+    const [error, setError] = useState(false);
+
+    const fetchData = async () => {
```

Comment:
When is this function called, I think you forgot to write a useEffect for it? It doesn't do anything atm (requst is commented out). Try to include only code that works in your pr.

Reply:
ok, I have added useeffect. But I leave this commented out code inside, I have to turn it up at the end of the week anyway